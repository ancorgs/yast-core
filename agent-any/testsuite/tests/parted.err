[yast2-agent-any] AnyAgentBasic.cc(parseNumber):136 *** not a number
[yast2-agent-any] AnyAgentBasic.cc(parseNumber):136 *** not a number
[yast2-agent-any] AnyAgentBasic.cc(parseNumber):136 *** not a number
