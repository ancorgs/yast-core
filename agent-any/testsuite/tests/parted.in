Disk geometry for /dev/sdb: 0.000-2063.830 megabytes
Disk label type: msdos
Minor    Start       End     Type      Filesystem  Flags
1          0.031   2063.034  primary   FAT         
