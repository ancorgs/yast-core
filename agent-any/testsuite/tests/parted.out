($["1":$["end":"2063.034", "filesystem":"FAT", "start":"0.031", "type":"primary"]])
